packages/lib-classifier/src/plugins/tasks/dropdown-simple/README.md

```diff
@@ -12,6 +12,9 @@ The Simple Dropdown Task for the Classifier is exactly what it says on the tin.
 
 ## Dev Notes
 
+**Limit Options** - Nov 2023
+- Full removal of "Other" option in the code as this code was refactored to use the `datalist` element. This can be re-added by modifying how the `updateAnnotation()` function works based on the configuration at a future point if it is needed/desired.
+
```

Comment:
The "other" option is still deleted from SimpleDropdownTask.js even though that component uses Grommet, not datalist. Is the deletion of "other" intentional? I think it's fine to delete the "other" option especially if the FEM lab does not support it's use. Please update this Readme accordingly though.